src/api/dimensions.js

```diff
@@ -99,6 +100,28 @@ export const dataElementGroupsQuery = {
     }),
 }
 
+export const apiFetchItemsByDimension = ({
+    engine,
+    dimensionId,
+    searchTerm,
+    pageSize,
+    page,
+}) => {
+    const query = {
+        dimensions: {
+            resource: `dimensions/${dimensionId}/items`,
+            params: objectClean({
```

Comment:
Is `objectClean` really needed?
For the `filters` in `api/dimensions` I used an empty array on which various filters are pushed only when needed.